### ISSUE TITLE
[Texture Request] Demon Dungeon Blocks/Items

### ISSUE THREAD

**Issue by WayofTime** [WayofTime/BloodMagic#878]
@Yulife 

The plan is to create dungeons that are generated from infusing the land with Demon Will. Each dungeon will be composed of blocks that are representative of that type of Will. If you want to think of it this way, the blocks are being infused with Will and converted.

Because there are five different types of Will, essentially we'd have five different sets of blocks. That does mean there will be a lot of textures - I am not expecting them right away, so take your time. I would like them in a month if possible, though.

For the overall feel, I'd like them to have a sort of... crystalline feel. If this isn't fully realizable for the given texture, that is fine, but since Will when solidified normally takes the form of a crystal I'd like to try to represent that in the block.

I'd like:
- [x] Stone
- [x] Stone brick
- [x] Lamp/light source, similar to glowstone but more stylized. Essentially a full block that gives off light.
- [x] Pillar block - would need two textures, top/bottom and sides
- [x] Stone bricks with different patterns - as many or as few as you want, but this is mainly to add building variety
- [ ] Log for an infused tree
- [ ] Leaves for an infused tree
- [ ] Sapling texture
- [ ] Demon Fruit texture - this is a fruit that would drop from the tree, preferably looking different for each type.

EDIT:
- [ ] Demon Ore - I am thinking an ore of a special demonic alloy that can be harvested. This ore won't spawn naturally - you will have to infuse nearby ores with Will, or inject it into the stone, or summon a modified meteor on your fayce.
- [ ] Paired with this, and with the metal blocks since it will be helpful, Demon Alloy Ingots. Also perhaps Demon Alloy Nuggets/Clusters - not quite vanilla nuggets, but bigger clusters that are obviously not pure. Dirty ore that has to be refined.
- [x] Fun three-layer pillars - These are the pillar beacons that we discussed earlier. They can be as big as you want, though I think your idea of having a three-layer pillar (stationary bottom block with a middle block that looks like it has a rotating component coupled with a free floating and rotation top block) is good. These will be a central element to the mod's progression!

There should be one of these for each type of Will (Raw, Corrosive, Destructive, Vengeful, Steadfast). The colours of the blocks should be a similar shade to the main colour that we have for each of the Will, but the blocks in the same category do not have to look like a simple pallet swap between the different types. For instance, the look of the stone bricks for Corrosive can be entirely different from the stone bricks for Steadfast. 

These will be one of the central gating mechanisms of the mod, so they will be viewed a lot and probably used for decoration. Thanks for everything! <3

**Comment by WayofTime** [comment]
Also, I noted it in IRC, but if there are other "pretty" blocks you want to add for decoration this is a good opportunity.

**Comment by Yulife** [comment]
Man, that's a lot of stuff. I have some cool ideas. Not sure about the crystalline style, if it's too bright it will be an eye sore. Maybe a few features. What's with the trees..?

**Comment by WayofTime** [comment]
Yeah, it is a lot of stuff, but I do think it will be needed to give the best experience to the player. 

Now, the trees, I figured that if you infuse Demon Will into plant life it would also have interesting effects. The demons have to have some sort of native vegetation, I felt.

**Comment by Yulife** [comment]
What will they do

**Comment by WayofTime** [comment]
Mainly decoration to start off with -  but the dungeons themselves will be to collect resources to furyher yourself in the mod.

**Comment by WayofTime** [comment]
I should also note that I'll add things like stairs and slabs with a lot of the textures. The main thing I want to work on is to have all of the resources that we will need for all of the t5 and even t6 things. Once we have resources I can think of what good devices we can make with them. Though that is going to be for when T1-T4 is well implemented.

**Comment by WayofTime** [comment]
Added two things that might help for future stuff and things.

**Comment by WayofTime** [comment]
@Yulife Added a few more things to the list. The ores/ingots/clusters are new, but the last item is just putting down in writing what we were discussing earlier.

**Comment by Yulife** [comment]
IMO it should be with the Ritual of the Meteor or however its called :p

**Comment by WayofTime** [comment]
Yeah, that's my plan for how you will -initially- get the Will ore/alloy. Then you can find an expensive Will-wise way to generate the ores, or use the expensive LP-wise way (meteor). Or find the alloy in the dungeons, but it won't be THAT easy! >:3

**Comment by wrincewind** [comment]
All complete by 1.21,with the slight exception of will-infused trees...
